dev-lang/fpc/fpc-3.0.4.recipe

```diff
@@ -0,0 +1,147 @@
+SUMMARY="Professional Pascal compiler"
+DESCRIPTION="Free Pascal is a 32, 64 and 16 bit professional Pascal compiler. \
+It can target many processor architectures: Intel x86 (including 8086), AMD64\
+/x86-64, PowerPC, PowerPC64, SPARC, ARM, AArch64, MIPS and the JVM. \
+Supported operating systems include Linux, FreeBSD, Haiku, Mac OS X/iOS/\
+iPhoneSimulator/Darwin, DOS (16 and 32 bit), Win32, Win64, WinCE, OS/2, \
+MorphOS, Nintendo GBA, Nintendo DS, Nintendo Wii, Android, AIX and AROS. \
+Additionally, support for the Motorola 68k architecture is available in the \
+development versions."
+HOMEPAGE="https://freepascal.org/"
+COPYRIGHT="1993-2015 Florian Klaempfl and others"
+LICENSE="
+	GNU LGPL v3
+	GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://sourceforge.net/projects/freepascal/files/Source/3.0.4/fpcbuild-3.0.4.tar.gz"
+CHECKSUM_SHA256="f66514e6f2c2e4e1bccccb4d554c24b77682ed61c87811ae5dd210f421855e76"
+SOURCE_URI_2="https://sourceforge.net/projects/freepascal/files/Documentation/3.0.4/doc-pdf.tar.gz#noarchive"
```

Comment:
Suggestion:
```diff
-SOURCE_URI="https://sourceforge.net/projects/freepascal/files/Source/3.0.4/fpcbuild-3.0.4.tar.gz"
+SOURCE_URI="https://downloads.sourceforge.net/freepascal/fpcbuild-$portVersion.tar.gz"
 CHECKSUM_SHA256="f66514e6f2c2e4e1bccccb4d554c24b77682ed61c87811ae5dd210f421855e76"
-SOURCE_URI_2="https://sourceforge.net/projects/freepascal/files/Documentation/3.0.4/doc-pdf.tar.gz#noarchive"
+SOURCE_URI_2="https://sourceforge.net/projects/freepascal/files/Documentation/$portVersion/doc-pdf.tar.gz#noarchive"
```
(We can use the redirector service for the main tarball, but we can't do that for the second tarball, `doc-pdf.tar.gz`, because its name does not include a `3.0.4` so when a new version gets released we would no longer get the pdf for this one.)